.github/workflows/code_formatting.yml

```
@@ -1,14 +1,29 @@
 name: Check code formatting
 
-on: [push, pull_request]
+on:
+   push:
+     branches:
+       - master
+     paths:
+       - "**/*.c"
+       - "**/*.cpp"
+       - "**/*.h"
+       - "**/*.hpp"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 11: did not find expected key

(syntax-check)